app/models/simulation_run_factory.rb

```diff
@@ -10,6 +10,10 @@ def rollback!
     self.save
     self
   end
+
+  def tmp_result
+    self.tmp_results_list.last['result'] unless self.tmp_results_list.blank?
```

Comment:
Co się dzieje w przypadku, jak ktoś ma jeszcze simulation runy z tmp_result, a używany będzie nowy kod? W przypadku tmp_results_list.blank? powinniśmy zwrócić attributes[:tmp_result] (jeśli będzie nilem to po prostu pójdzie nil).